action.yml

```diff
@@ -1,57 +1,61 @@
-name: Semantic Versioning
-on:
-  workflow_call:
-    # inputs:
-      # enable-code-scans:
-        # required: true
-        # type: boolean
-    secrets:
-      pat:
-        required: true
-        description: Github Personal Access token needed to execute authorized actions (e.g creating a commit)
+name: 'Semantic Versioning'
+description: 'Calculates the new semantic version based on pull request labels and then updates the version according to that'
```

Comment:
I would add `and based of the lastest prerelease version`

Reply:
fixed